### PR TITLE
Added secured connection sample for TCP client

### PR DESCRIPTION
**Pull request by oskardudycz**
Added secured connection sample to a single node cluster. This is done accordingly as for the gRPC client: https://github.com/EventStore/EventStore-Client-Dotnet/tree/master/samples/secure-with-tls

### CODE REVIEW

**Review by YoEight — APPROVED** [review]
Tested on my machine.

File: samples/secure-with-tls/README.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+## Overview
+
+The sample shows how to run the .NET TCP client secured by TLS certificates.
+
+Read more in the docs:
+- [Security](https://developers.eventstore.com/server/v20/server/security/)
+- [Running EventStoreDB with `docker-compose`](https://developers.eventstore.com/server/v20/server/installation/docker.html#use-docker-compose)
+- [Event Store Certificate Generation CLI](https://github.com/EventStore/es-gencert-cli)
+
+It is essential for production use to configure EventStoreDB security features to prevent unauthorised access to your data.
```
Comment (ylorph) [inline samples/secure-with-tls/README.md:25]: TLS prevents eavesdropping, not unauthorized access as we secure the communication channel only

File: samples/secure-with-tls/README.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+EventStoreDB supports TCP with TLS and SSL. Each protocol has its security configuration, but you can only use one set of certificates for TLS and HTTPS.
```
Comment (ylorph) [inline samples/secure-with-tls/README.md:26]: need to verify this...
As far as I understand, TLS superseeds SSL , HTTPS is secured by TLS not SSL

File: samples/secure-with-tls/README.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+### Certificates
+
+The protocol security configuration depends a lot on the deployment topology and platform. We have created an interactive [configuration tool](https://github.com/EventStore/es-gencert-cli), which also has instructions on generating and installing the certificates and configure EventStoreDB nodes to use them.
```
Comment (ylorph) [inline samples/secure-with-tls/README.md:30]: link to configurator.eventstore.com as well ?

**Pull request merged**